src/pages/instances/CreateInstanceForm.tsx

```diff
@@ -106,6 +106,9 @@ const CreateInstanceForm: FC = () => {
       .matches(/^[A-Za-z0-9-]+$/, {
         message: "Only alphanumeric and hyphen characters are allowed",
       })
+      .matches(/^[^0-9].*$/, {
+        message: "Name cannot start with a number",
```

Comment:
Can this be renamed to instance name? feels a bit more explicit to me.
```suggestion
        message: "Instance name cannot start with a number",
```

Reply:
Good idea. I also changed the condition below, because it must start with a letter, - is also an invalid start.